numpy/lib/function_base.py

```diff
@@ -1337,8 +1337,8 @@ def interp(x, xp, fp, left=None, right=None, period=None):
     --------
     scipy.interpolate
 
-    Notes
-    -----
+    Warnings
```

Comment:
We don't use it a lot, but this is a valid section header in numpydoc, and in this case I agree it's warranted.

Reply:
Isn't this section header supposed to be about `Warning` objects emitted via the `warnings` module?

Reply:
The numpydoc description of this section is _"An optional section with cautions to the user in free text/reST."_.